src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -185,7 +291,91 @@ read_file_itk(std::string filename)
       error(ex.what());
       return 0;
     }
+}
+
+//To read any file format via ITK
+template<>
+STIRImageMulti*
+read_file_itk(std::string filename)
+{
+#ifndef NDEBUG
+  info(boost::format("Opening a multi image: %1%") % filename);
+#endif
+
+  typedef itk::GDCMImageIO       ImageIOType;
+  ImageIOType::Pointer dicomIO = ImageIOType::New();
+
+  try
+    {
+      // We can't deal with DICOMs
+      if (dicomIO->CanReadFile(filename.c_str())) {
```

Comment:
DICOM doesn't define deformation types etc.
we could stick to a single template definition but doing a check on the type, e.g. via `GetPixelType` presumably)